tests/test_functional.py

```diff
@@ -801,3 +801,28 @@ def test_obal_verify_tag_all():
         'koji list-pkgs --quiet --tag obaltest-nightly-rhel7'
     ]
     assert_mockbin_log(expected_log)
+
+@obal_cli_test(repotype='copr')
+def test_copr_project_one_chroot():
+    assert_obal_success(['copr-project', 'foreman'])
+
+    expected_log = [
+        "copr-cli create example/foreman --description foreman --chroot epel-7-x86_64 --unlisted-on-hp on",  # noqa: E501
+        "copr-cli edit-chroot example/foreman/epel-7-x86_64 --repos http://mirror.centos.org/centos/7/sclo/x86_64/rh/ --packages 'rh-ruby24-build scl-utils-build'",  # noqa: E501
+
+    ]
+    assert_mockbin_log(expected_log)
+
+
+@obal_cli_test(repotype='copr')
+def test_copr_project_many_chroots():
+    assert_obal_success(['copr-project', 'client'])
+
+    expected_log = [
+        "copr-cli create example/foreman-client --description foreman-client --chroot rhel-9-x86_64 --chroot rhel-8-x86_64 --chroot rhel-7-x86_64 --unlisted-on-hp on",  # noqa: E501
+        "copr-cli edit-chroot example/foreman-client/rhel-9-x86_64",
+        "copr-cli edit-chroot example/foreman-client/rhel-8-x86_64",
+        "copr-cli edit-chroot example/foreman-client/rhel-7-x86_64"
```

Comment:
it's a bit weird that there are *no* edit commands at the end, but it's a valid (even if unnecessary) invocation.

IMHO something we should optimize, but not blocking

Reply:
I do not follow what you mean here.

Reply:
`copr-cli edit-chroot example/foreman-client/rhel-7-x86_64` is a noop as it does not pass any options to change (as there are no custom settings defined for that chroot).

I think it can be optimized away, but not necessarily today.

Reply:
Oh I see your point: not only is this is a bad test example, it actually highlights an inefficiency if there is no edits to perform.